src/js/components/utils/colors.js

```diff
@@ -15,9 +15,22 @@ export const colorForName = (name, theme) => {
   return color;
 };
 
-export const colorIsDark = (color) => {
+function parseHexToRGB(color) {
   // https://stackoverflow.com/a/42429333
-  const [red, green, blue] = color.match(/[A-Za-z0-9]{2}/g).map(v => parseInt(v, 16));
+  return color.match(/[A-Za-z0-9]{2}/g).map(v => parseInt(v, 16));
+}
+
+function getRGBArray(color) {
+  if (color.startsWith('#')) {
+    return parseHexToRGB(color);
+  } else if (color.startsWith('rgb')) {
+    return color.match(/rgba?\((\s?[0-9]*\s?),(\s?[0-9]*\s?),(\s?[0-9]*\s?).*?\)/).splice(1);
```

Comment:
Should also allow `rgb(...)`

Reply:
This allows for rgb, notice the question mark after the `a`, which makes it optional. I've tested this regex with the following inputs:

* `rgb(50, 50, 50)` -> returns [50, 50 50]
* `rgba(50, 50, 50, 0.1)` -> returns [50, 50 50]
* `rgba( 50,50, 50,)` -> returns [50, 50 50]

The last one may have spaces before and after, I get that too.

Reply:
Ah. Thanks.